_positions/visual-designer.md

```diff
@@ -1,40 +1,29 @@
 ---
 title: Visual Designer
-permalink: who-we-are-hiring/positions/visual-designer/
+permalink: roles-and-teams/positions/visual-designer/
```

Comment:
See #155. Also, has this listing ever been active? If so, please add a redirect so the old URL won't break.